src/index.html

```diff
@@ -8,6 +8,59 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="rating">
+      <div class="rating__group stars--0">
```

Comment:
BEM modifier should only be used on element which has the corresponding block class - Checklist

you do not need additional wrapper